ingestion/transformers/assign_cadet_domains.py

```diff
@@ -0,0 +1,117 @@
+import json
```

Comment:
It's not super well documented by DataHub but my understanding is that sources create entities, while transformers alter entity creation requests to add more aspects. So conceptually it fits better to alter the creation of all the datasets from dbt with another domain association upon the creation of the entity. You get the entities for free as part of the dbt ingestion, otherwise you'd have to do a lookup in the domain creation source.

It could probably work all in the source file, and the manifest wouldn't have to be read twice. Maybe I can pull out the code to read from s3 into a module?

Reply:
Yep I think that makes sense, and extracting the common code would be a good improvement 👍🏻